camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .detail(ResultErrorBuilder.attribute("enum.values"), result.getEnumChoices(entry.getKey()))
+                        .detail("enum.values", result.getEnumChoices(entry.getKey()))
```

Comment:
yeah, thats a bit tricky. But having mixed cases (like here with 'enum.values') is no problem ?

lower case enums are indeed a bit cumbersome. at least my IDEA show them still in italics ;-)

Let me adapt the rest of camel and open a wider discussion on the 'official' PR